src/pages/pages.jsx

```diff
@@ -1,6 +1,7 @@
 import Home from "./Home/Home";
 import Playlist from "./Playlist/Playlist";
 import VideoListing from "./VideoListing/VideoListing";
+import VideoCardPage from "./VideoListing/VideoCardPage";
```

Comment:
For these also you can make a separate file to export from there

Reply:
It is already a separate file Which is importing and exporting